src/MasterMix/types.ts

```diff
@@ -0,0 +1,40 @@
+import { Modify } from '@mll-lab/js-utils';
+import { ColumnType } from 'antd/es/table';
+import { ReactNode } from 'react';
+
+export type Ingredient = {
+  key: number;
+  title: string | NonNullable<ReactNode>;
+  volume: number;
+};
+export type MasterMixProps = {
```

Comment:
We should export this to package users, as of now it would not be. Let's selectively reexport from `MasterMix/index.tsx` - but for example `PipettingLossTableColumn` does seem unnecessary.